homeassistant/components/zha/core/channels/measurement.py

```diff
@@ -57,7 +57,7 @@ class RelativeHumidity(ZigbeeChannel):
     REPORT_CONFIG = [
         {
             "attr": "measured_value",
-            "config": (REPORT_CONFIG_MIN_INT, REPORT_CONFIG_MAX_INT, 50),
+            "config": (REPORT_CONFIG_MIN_INT, REPORT_CONFIG_MAX_INT, 100),
```

Comment:
This value is specific for the Humidity channel, so it doesn't really make sense to move it to const.py IMO. Deltas in the other channels are also defined as literal numbers.
Regarding exposing this, I think it would be good to expose the report intervals in the UI, not yaml, but that would require a much bigger change. Until then, I think this PR brings a much needed fix (my sonoff temp. sensor drained its battery in 4 months).